src/__tests__/__snapshots__/Badge.js.snap

```diff
@@ -207,14 +207,16 @@ exports[`Badge renders 1`] = `
   font-size: 12px;
   font-weight: 600;
   text-transform: uppercase;
-  -webkit-letter-spacing: 0.05em;
-  -moz-letter-spacing: 0.05em;
-  -ms-letter-spacing: 0.05em;
-  letter-spacing: 0.05em;
+  -webkit-letter-spacing: 0.025em;
+  -moz-letter-spacing: 0.025em;
+  -ms-letter-spacing: 0.025em;
+  letter-spacing: 0.025em;
   padding-left: 8px;
   padding-right: 8px;
   padding-top: 4px;
   padding-bottom: 4px;
+  background-color: #f6f8fa;
+  color: #001833;
```

Comment:
Could the namespaces of colors be applied? Such as: `background-color: lightGray;` & `color: text;`?